native/src/components/EastereggImage.tsx

```diff
@@ -17,6 +17,10 @@ const ApiUrlText = styled.Text`
   color: red;
 `
 
+const FullOpacityPressable = styled(Pressable)`
```

Comment:
Normally we name these styled props just "styledPressable", why have you decided to do it different here?

Reply:
I thought styling in this case only has one sole purpose, i.e. preventing showing a feedback to the user in order not to indicate that it is pressable, so I thought naming this differently might add additional information. However, I guess that `FullOpacityPressable` doesn't really achieve that so I will just rename to `StyledPressable`.